### ISSUE TITLE
Add a `toggle` functionality similar to `IndentBlanklineToggle`

### ISSUE THREAD

**Issue by wimstefan** [lukas-reineke/virt-column.nvim#9]
From time to time I have to select & copy text with the mouse and I always [switch off all nice decorative elements](https://github.com/wimstefan/dotfiles/blob/master/config/nvim/init.lua#L151-L174) to get "plain" text for the copy process. 

It's not a big thing for sure - more a "nice to have " thingie ;)

**Comment by lukas-reineke** [comment]
You can turn it off by just doing `:set colorcolum=`
But yeah, maybe a command that toggles on and off with the old value would be nice.

**Comment by wimstefan** [comment]
Wow that was an easy one! Thanks for putting my nose into it 👍

**Comment by DanielKriz** [comment]
This workaround doesn't seem to work on virt-columns specified in plugins config (with `virtcolumn = {'values'}`). I tried some really dirty fix with setting `char = ''`, but that seems to break things. This toggle feature would be really nice.

**Comment by AlexSWall** [comment]
I agree that a toggle would be nice; changing the colour doesn't prevent it being including when copying the text from the buffer via a program/selection external to vim (e.g. cursor selection from terminal, tmux copy, etc.).

**Comment by andradei** [comment]
For what is worth, I did it without `colorcolumn` because I want that to be constant in my config. So I went for the lua customization approach:

This is overkill and hacky, but I enjoyed writing it and there is no noticeable performance hit.

```lua
-- when I setup virt-column I do:
local virt_column = require "virt-column"

-- This is me adding the `enabled` value here so it can be toggled in a keymap.
virt_column.enabled = true

local config = {
  enabled = virt_column.enabled,
  -- the rest of my configuration
}

virt_column.setup(config)

--------------------------------------------------------------------------

-- then in my lua/mappings.lua:

vim.keymap.set("n", "<leader>|", function()
  local vc = require "virt-column"
  -- The `enabled` key was added by me during setup so it can be toggled.
  vc.enabled = not vc.enabled -- toggle the boolean value
  vc.update { enabled = vc.enabled } -- update the plugin config
  vim.api.nvim_command "redraw!" -- redraw the screen to make the changes take effect
end, { desc = "toggle vertical guide/ruler" })
```
